app/shared/package.json

```diff
@@ -8,8 +8,10 @@
     "@kiwicom/mobile-localization": "^0",
     "@kiwicom/react-native-native-modules": "^0.1.25",
     "@ptomasroos/react-native-multi-slider": "^0.0.12",
+    "react-native-fetch-blob": "^0.10.8",
     "react-native-image-progress": "^1.1.0",
     "react-native-modal": "^5.4.0",
+    "react-native-pdf": "^3.0.8",
```

Comment:
This can be probably removed now.

Reply:
I forgot, it should be in pdf package.json yes